routes/home.js

```diff
@@ -2,7 +2,11 @@ const apps = require('../lib/apps')
 
 module.exports = (req, res) => {
   const context = Object.assign(req.context, {
-    apps: apps
+    apps: apps,
+    page: {
+      title: 'Electron | Build cross platform desktop apps with JavaScript, HTML, and CSS.',
```

Comment:
This page title and description should live in data/locale.yml

Reply:
I will move all it to locale.yml